chaoskube/chaoskube_test.go

```diff
@@ -540,3 +552,103 @@ func (t ThankGodItsFriday) Now() time.Time {
 	blackFriday, _ := time.Parse(time.RFC1123, "Fri, 24 Sep 1869 15:04:05 UTC")
 	return blackFriday
 }
+
+func (suite *Suite) TestMinimumAge() {
+	type pod struct {
+		name         string
+		namespace    string
+		creationTime time.Time
+	}
+
+	for _, tt := range []struct {
+		minimumAge time.Duration
+		now        func() time.Time
+		pods       []pod
+		candidates int
+	}{
+		// no minimum age set
+		{
+			time.Duration(0),
+			func() time.Time { return time.Date(0, 10, 24, 10, 00, 00, 00, time.UTC) },
+			[]pod{
+				{
+					name:         "test1",
+					namespace:    "test",
+					creationTime: time.Date(0, 10, 24, 9, 00, 00, 00, time.UTC),
+				},
+			},
+			1,
+		},
+		// minimum age set, but pod is too young
+		{
+			time.Hour * 12,
```

Comment:
Thanks for the tests! For the sake of simplicity can we change the `minimumAge` values to something smaller like `1h` or so? I think it will be easier to understand them then.